app/models/mixins/relationship_mixin.rb

```diff
@@ -565,20 +617,22 @@ def add_children(*child_objs)
   alias_method :add_child, :add_children
 
   def parent=(parent)
+    return call_ancestry_method(:parent=, parent) if use_ancestry?
```

Comment:
this needs the argument.

But I don't think the other calls to `call_ancestry_method` need it